tsdb/README.md

```diff
@@ -33,16 +33,15 @@ cpu,host=server\ A,region=us\ west
 
 ## Fields
 
-Fields are are values associated with the measurement.  Every line must have at least one field.  Multiple fields should be separated with commas and not spaces in between.
+Fields are key-value metrics associated with the measurement.  Every line must have at least one field.  Multiple fields must be separated with commas and not spaces.
 
-Fields can be one of four types.  The value written for a given field defines the type of the field.
+Field keys are always strings and follow the same syntactical rules as described above for tag keys and values. Field values can be one of four types.  The first value written for a given field on a given measurement defines the type of that field for all series under that measurement.
```

Comment:
Right now this behaviour is an intrinsic characteristic of the design. There are no plans to change this behaviour (though it is possible).

I agree it's complicated to explain. A simple explanation is probably best, since it covers what people should really be doing.